npm/grep/cypress/e2e/unit.js

```diff
@@ -137,15 +151,17 @@ describe('utils', () => {
 
     it('filters out empty tags', () => {
       const parsed = parseTagsGrep(',, @tag1,-@tag2,, ,, ,')
+
       expect(parsed).to.deep.equal([
         [{ tag: '@tag1', invert: false }],
         [{ tag: '@tag2', invert: true }],
       ])
     })
 
-    // would need to change the tokenizer
+    // TODO: would need to change the tokenizer
```

Comment:
This hasn't changed in quite a while but it's now getting linted so i added the TODO